packages/firebase_vertexai/firebase_vertexai/lib/src/error.dart

```diff
@@ -56,12 +56,16 @@ final class ServiceApiNotEnabled implements VertexAIException {
       'The Vertex AI in Firebase SDK requires the Vertex AI in Firebase API '
       '(`firebasevertexai.googleapis.com`) to be enabled in your Firebase project. Enable this API '
       'by visiting the Firebase Console at '
-      'https://console.firebase.google.com/$_projectId/genai '
+      'https://console.firebase.google.com/project/${_getID()}/genai '
       'and clicking "Get started". If you enabled this API recently, wait a few minutes for the '
       'action to propagate to our systems and then retry.';
 
   @override
   String toString() => message;
+
+  String _getID() {
```

Comment:
Either `get _id` or `_parseId()` is preferred over a name staring with "get". I'd go with the first one.
https://dart.dev/effective-dart/design#avoid-starting-a-method-name-with-get

```suggestion
      'https://console.firebase.google.com/project/$_id/genai '
      'and clicking "Get started". If you enabled this API recently, wait a few minutes for the '
      'action to propagate to our systems and then retry.';

  @override
  String toString() => message;

  String get _id {
```